addons/mail/static/src/discuss_app/channel_selector.js

```diff
@@ -20,17 +21,43 @@ export class ChannelSelector extends Component {
         this.store = useStore();
         this.threadService = useState(useService("mail.thread"));
         this.personaService = useService("mail.persona");
-        this.orm = useService("orm");
+        this.sequential = useSequential();
+        this.orm = this.env.services.orm;
         this.state = useState({
             value: "",
             selectedPartners: [],
+            navOption: {
```

Comment:
Might want to rename this key in order to reflect its use? (props of the navigable list which is not easy to get at first glance)